docs/tutorial.md

```diff
@@ -33,29 +41,46 @@ Namespace also supports access by label and IRI checking.
 Both of these features requires loading an ontology.
 The following example shows how to create an EMMO namespace with IRI checking.
 The keyword argument `label_annotations=True` enables access by `skos:prefLabel`, `rdfs:label` or `skos:altLabel`.
+What labels to use can also be specified explicitly.
 The `check=True` enables checking for existing IRIs.
-The `triplestore_url=...` is a resolvable URL that can be read by the 'rdflib' backend.
-It is needed, because the 'rdflib' backend is currently not able to load EMMO from the `http://emmo.info/emmo#` namespace.
 
 ```python
 >>> EMMO = ts.bind(
-...     "emmo", "http://emmo.info/emmo#",
+...     "emmo", "https://w3id.org/emmo#",
 ...     label_annotations=True,
 ...     check=True,
-...     triplestore_url="https://emmo-repo.github.io/versions/1.0.0-beta4/emmo-inferred.ttl",
 ... )
```

Comment:
This is a nice feature that is good to document in this simple example.
Don't remove it completely

Reply:
It is not removed. I only removed the `triplestore_url` argument, since it is not needed any more. All the labels are now fetch from the IRI "https://w3id.org/emmo#" provided as the second argument. 

Enabling simplifications like this, is part of the goal of this PR :-)

Reply:
It is removed in the tutorial. I think it should stay, either in the same example or add another just after to point out this very important possiblity.

Reply:
Added a reformulation of the comment about the use of the `triplestore` argument below the code example.